plugins/modules/azure_rm_containerregistrytag.py

```diff
@@ -0,0 +1,349 @@
+            if not self.check_mode:
+                self.delete_repository(self.repository_name)
+        else:
+            self.log("no action")
+            self.results["changed"] = False
+
+        return self.results
+
+    def get_client(self):
+        registry_endpoint = self.registry if self.registry.endswith(".azurecr.io") else self.registry + ".azurecr.io"
+        return ContainerRegistryClient(
+            endpoint=registry_endpoint,
+            credential=self.azure_auth.azure_credential_track2,
+            audience="https://management.azure.com",
+        )
+
+    def get_repository(self, repository_name):
+        response = None
+        try:
+            response = self._client.get_repository_properties(repository=repository_name)
+            self.log(f"Response : {response}")
+        except Exception as e:
+            self.log(f"Could not get ACR repository for {repository_name} - {str(e)}")
+
+        if response is not None:
+            return response.name
+
+        return None
+
+    def get_tag(self, repository_name, tag_name):
+        response = None
+        try:
+            self.log(f"Getting tag for {repository_name}:{tag_name}")
+            response = self._client.get_tag_properties(repository=repository_name, tag=tag_name)
+            self.log(f"Response : {response}")
+        except Exception as e:
+            self.log(f"Could not get ACR tag for {repository_name}:{tag_name} - {str(e)}")
+
+        return response
+
+    def import_tag(self, repository, tag, resource_group, registry, source_image):
+        source_tag = get_tag(source_image["repository"], source_image["name"])
+        dest_repo_name = repository if repository else source_image["repository"]
+        dest_tag_name = tag if tag else source_image["name"]
+        dest_tag = get_tag(dest_repo_name, dest_tag_name)
+        creds = None if not source_image["credentials"] else ImportSourceCredentials(
+            username=source_image["credentials"]["username"],
+            password=source_image["credentials"]["password"],
+        )
+        params = ImportImageParameters(
+            target_tags=[dest_tag],
+            source=ImportSource(
+                registry_uri=source_image["registry_uri"],
+                source_image=source_tag,
+                credentials=creds,
+            )
+        )
+        try:
+            if not resource_group:
+                resource_group = self.get_registry_resource_group(registry)
+
+            self.log(f"Importing {source_tag} as {dest_tag} to {registry} in {resource_group}")
+            poller = self.containerregistry_client.registries.begin_import_image(resource_group_name=resource_group,
+                                                                                 registry_name=registry,
+                                                                                 parameters=params)
+            self.get_poller_result(poller)
```

Comment:
Can we return this result to result? We can determine the resource configuration created by returning information. Thank you very much!